internal/controller/impersonatorconfig/impersonator_config.go

```diff
@@ -363,6 +379,7 @@ func (c *impersonatorConfigController) serviceExists(serviceName string) (bool,
 		return false, nil, nil
 	}
 	if err != nil {
+		// untested
```

Comment:
Do we want all of these "untested" comments to linger in the production code?

Reply:
Same question. If we were to put that comment on every line of untested code in the project, that would be a lot of distracting comments. :)

Reply:
I'll remove with the next push